spec/features/idv/doc_auth/send_link_step_spec.rb

```diff
@@ -30,11 +31,14 @@
     expect(page).to have_current_path(idv_doc_auth_link_sent_step)
   end
 
-  it 'sends a link that does not contain any underscores' do
-    # because URLs with underscores sometimes get messed up by carriers
+  it 'sends a link that matches expected URL structure' do
     expect(Telephony).to receive(:send_doc_auth_link).and_wrap_original do |impl, config|
+      # because URLs with underscores sometimes get messed up by carriers
       expect(config[:link]).to_not include('_')
```

Comment:
one argument against naming the key `session_id` (or anything longer) is that these URLs have a length limit of 160 characters because sometimes our messages are broken up across multiple SMS messages. What's the path length of this with the session UUID?

Reply:
> one argument against naming the key `session_id` (or anything longer) is that these URLs have a length limit of 160 characters because sometimes our messages are broken up across multiple SMS messages. What's the path length of this with the session UUID?

Yeah, it's a good question. Trying to get some hard numbers here:

Today (210 characters):

>You've requested to verify your identity on a mobile phone. Please take a photo of your state issued ID: https://secure.login.gov/verify/capture-doc/mobile-front-image?token=00000000-0000-0000-0000-000000000000

With `session`, to `document_capture` (255 characters):

>You've requested to verify your identity on a mobile phone. Please take a photo of your state issued ID: https://secure.login.gov/verify/capture-doc/document-capture?token=00000000-0000-0000-0000-000000000000&session=00000000-0000-0000-0000-000000000000

Noting above that even today we're already exceeding 160 characters, though based on my own experience, and [backed up by Twilio's documentation](https://www.twilio.com/docs/glossary/what-sms-character-limit), it's likely that my phone / carrier is rebuilding this to appear as a single message.

So it might not be a problem? Or at least no more a problem than we already have?

It does motivate to push toward one of the options mentioned in my original comment to try to consolidate to one or the other of `token` or `session`, though each of these come with their own challenges:

- If we consolidate to `token`, we need to have a way to associate `doc_captures` entry to a `document_capture_sessions` entry
- If we consolidate to `session`, we need to reimplement token expiration handling

Reply:
It's definitely been a problem for some combination of carriers and phones in the past: https://cm-jira.usa.gov/browse/LG-3154

Maybe we can come up with a single-letter path or something as a way to shorten the URL? 

But anyways, ya lets go ahead with it as `session` in the URL for now and we can see if we get any bug reports?

Reply:
This is slightly improved as of the rebased b5f1334ea68533771810bbf9b83022da279d3482:

- We only include one or the other of `token` or `document-capture-session` parameters
- The base URL path is shortened from `/verify/capture-doc/mobile-front-image` to `/verify/capture-doc`